### PR TITLE
stdv dev added

### PR DESCRIPTION
**Pull request by Ads7**

### CODE REVIEW

File: RequestController.js [hunk 1]
```diff
@@ -50,7 +50,6 @@ function singleApproachView( segments, response, postData ) {
     else {
     }
 }
-
```
Comment (sahebpreet) [inline RequestController.js]: can you remove these changes which are not related?

File: datacontroller/zApproach/AppController.js [hunk 2]
```diff
@@ -15,10 +15,11 @@ function updateZValues( options ) {
     scores.forEach( function( score ) {
 
         if ( score['mean'] ) {
-
+            // var oldmean=score.mean;
```
Comment (sahebpreet) [inline datacontroller/zApproach/AppController.js]: and also these changes

**Comment by Ads7** [comment]
removed unrelated changes

File: datacontroller/zApproach/AppController.js [hunk 3]
```diff
@@ -15,10 +15,11 @@ function updateZValues( options ) {
...
             score[ 'mean' ] = ( score.total * score.mean + score.count )/( score.total + 1 );
             score[ 'variance' ] = Math.sqrt( ( score.total * score.variance * score.variance + score.count * score.count ) / ( score.total + 1 ) );
             score[ 'total' ]++;
+            //score['std-dev'] = Math.sqrt(((score.count-score.mean) + (score.total - 1) * (oldmean - score.mean)/(score.total - 1))
```
Comment (sahebpreet) [inline datacontroller/zApproach/AppController.js]: some changes are also present now like this

File: datacontroller/zApproach/AppController.js [hunk 4]
```diff
@@ -59,4 +60,4 @@ function fetchTrends( options ) {
 exports.updateScores = updateScores;
 exports.changeConstants = changeConstants;
 exports.handleData = handleData;
-exports.fetchTrends = fetchTrends;
\ No newline at end of file
+exports.fetchTrends = fetchTrends;
```
Comment (sahebpreet) [inline datacontroller/zApproach/AppController.js]: and this

File: datasource/live streamingWrapper.js [hunk 5]
```diff
@@ -11,7 +11,7 @@ var twit = new twitter({
 
 twit.stream('statuses/filter', {'locations':'-122.75,36.8,-121.75,37.8,-74,40,-73,41'}, function(stream) {
   stream.on('data', function (data) {
-    console.log(data);
+    //console.log(data);
```
Comment (sahebpreet) [inline datasource/live streamingWrapper.js]: this too

File: index.js [hunk 6]
```diff
@@ -8,6 +8,7 @@ var handle = {
     "approach" : "./RequestController",
     "static" : "./RequestController",
     "fetch_trends" : "./RequestController"
+
```
Comment (sahebpreet) [inline index.js]: and also this we should not record redundant changes, for clean git history
